modules/grizzly/src/main/java/org/glassfish/grizzly/nio/transport/TCPNIOConnectorHandler.java

```diff
@@ -431,4 +431,99 @@ public void failed(Throwable throwable) {
             return needFuture ? ReadyFutureImpl.<Connection>create(e) : null;
         }
 	}
+
+    @Override
+    public FutureImpl<Connection> connectAsyncForRedirect(SocketAddress remoteAddress, SocketAddress localAddress,
+                                                  CompletionHandler<Connection> completionHandler, boolean needFuture,
+                                                  boolean onlyAddCompletionHandlerToFuture) {
+        final TCPNIOTransport nioTransport = (TCPNIOTransport) transport;
```

Comment:
this looks copy pasted from somewhere else. Why can't be reuse something already in grizzly?